src/ubuntu/manifest.json

```
@@ -651,6 +651,23 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm64",
+              "dockerfile": "src/ubuntu/22.04/",
```

Nit: just for consistency

```suggestion
              "dockerfile": "src/ubuntu/22.04",
```